PKGBUILD

```diff
@@ -221,10 +214,6 @@ build() {
   _ungoogled_repo="$srcdir/$pkgname-$_uc_ver"
   readarray -t -O ${#_flags[@]} _flags < "${_ungoogled_repo}/flags.gn"
 
-  # See https://github.com/ungoogled-software/ungoogled-chromium-archlinux/issues/123
-  CFLAGS="-march=x86-64 -mtune=generic -O2 -pipe -fno-plt"
-  CXXFLAGS="$CFLAGS"
-
```

Comment:
hm I'm not entirely sure if https://github.com/ungoogled-software/ungoogled-chromium-archlinux/issues/123 is still relevant

Reply:
This shouldn't be an issue anymore as the problematic flags are already excluded. It compiles and runs fine on my end.

Reply:
well lets see how it goes